pkg/build/registry/buildlog/rest.go

```diff
@@ -120,6 +121,8 @@ func (r *REST) waitForBuild(ctx kapi.Context, build *api.Build) error {
 	done := make(chan struct{})
 	errchan := make(chan error)
 	go func(ch <-chan watch.Event) {
+		defer util.HandleCrash()
```

Comment:
This fix seems a little random and unrelated to the deployment logs PR- what's the story?